js/layout-javascript/agenda-code.js

```diff
@@ -2401,6 +2409,27 @@ DynamicList.prototype.addDetailViewData = function(entry) {
       content = entry.originalData[dynamicDataObj.column];
     }
 
+    if (dynamicDataObj.type === 'image') {
+      content = entry.originalData[dynamicDataObj.column];
+
+      if (!Array.isArray(content)) {
```

Comment:
Let's check strings specifically because `.split()` only works with strings.

```suggestion
      if (typeof content === 'string') {
```